src/measure_theory/integral/circle_integral.lean

```diff
@@ -129,6 +129,17 @@ funext $ λ θ, circle_map_eq_center_iff.2 rfl
 lemma circle_map_ne_center {c : ℂ} {R : ℝ} (hR : R ≠ 0) {θ : ℝ} : circle_map c R θ ≠ c :=
 mt circle_map_eq_center_iff.1 hR
 
+lemma circle_map_ne_on_ball {R : ℝ} (hR : 0 < R) {z w : ℂ} (hw : w ∈ ball z R) :
+  ∀  (x : ℝ), circle_map z R x - w ≠ 0 :=
```

Comment:
```suggestion
  ∀ (x : ℝ), circle_map z R x - w ≠ 0 :=
```
(I already pointed out this super-trivial fix before -- if you use the built-in github feature to automatically apply review suggestions as a commit, then it saves a lot of effort and also avoids the risk of overlooking small fixes like this.)